src/cli/env.js

```diff
@@ -1,5 +1,9 @@
 const parseEnv = () => {
-    // Write your code here 
+    for (const prop in process.env) {
+        if (prop.includes('RSS_')) {
+            console.log(`${prop}=${process.env[prop]}`)
```

Comment:
you have to print in this format RSS_name1=value1; RSS_name2=value2 
in one line and with semicolon as separator